src/lib.rs

```diff
@@ -33,12 +33,126 @@ impl<'a> Image<'a> {
             format,
         }
     }
+
+    fn get_alpha(&self, x: usize, y: usize) -> f32 {
+        self.pixels[x * 4 + 3 + y * 4 * self.width as usize] as f32 / 255.0
```

Comment:
I am not sure this is right, or at least I think it could be written better:
- `(y * self.width + x)` gives us the pixel index
- Multiplying by 4 takes into account 4-byte stride each pixel has.
- Adding 3 fetches the alpha of the pixel.

I think `(y * self.width + x) * 4 + 3` makes it clearer what is what. I would also making some of these into separate variables for additional clarity.

Reply:
Yeah that's definitely clearer, will change. I don't think it's wrong currently, just written badly